system/Database/Postgre/Connection.php

```diff
@@ -253,7 +254,7 @@ public function escape($str)
     /**
      * Platform-dependant string escape
      */
-    protected function _escapeString(string $str): string
+    protected function _escapeString(string|Stringable $str): string
```

Comment:
Same question for `pg_escape_string`.

I think we need to add a string cast to both function calls.

Reply:
Thanks for the good insight!